source/reference/mongotranslate.txt

```diff
@@ -51,33 +53,38 @@ Command Line Options
 
 .. include:: /includes/option/option-mongotranslate-schema.rst
 
+.. include:: /includes/option/option-mongotranslate-dbName.rst
+
 .. include:: /includes/option/option-mongotranslate-format.rst
 
 .. include:: /includes/option/option-mongotranslate-explain.rst
 
 .. include:: /includes/option/option-mongotranslate-mongoversion.rst
 
-.. include:: /includes/option/option-mongotranslate-dbName.rst
-
 Examples
 --------
```

Comment:
I don't have the dataset that you are using but I think there is something wrong. --dbname is not provided and the full namespace <database>.<collection> is not used. I think we'll need to do one or the other?